File: skills/astra-toolkit/clients/typescript/examples/tables-drop-table-name.ts
import { DataAPIClient } from "@datastax/astra-db-ts";

// Get a database
const client = new DataAPIClient();
const database = client.db(process.env.ASTRA_DB_API_ENDPOINT!, {
  token: process.env.ASTRA_DB_APPLICATION_TOKEN!,
});

// Drop a table
(async function () {
  await database.dropTable("**TABLE_NAME**");
})();

File: skills/astra-toolkit/clients/python/examples/tables-drop-table-name.py
import os
from astrapy import DataAPIClient

# Get a database
client = DataAPIClient()
database = client.get_database(
    os.environ["ASTRA_DB_API_ENDPOINT"], token=os.environ["ASTRA_DB_APPLICATION_TOKEN"]
)

# Drop a table
database.drop_table("**TABLE_NAME**")
